spec/index.bs

```diff
@@ -5972,7 +5972,8 @@ Issue: convert "Valid Texture Copy Range" into an algorithm with parameters, sim
                     - [$validating GPUImageCopyTexture$](|destination|, |copySize|) returns `true`.
                     - |dstTextureDesc|.{{GPUTextureDescriptor/usage}} contains {{GPUTextureUsage/COPY_DST}}.
                     - |srcTextureDesc|.{{GPUTextureDescriptor/sampleCount}} is equal to |dstTextureDesc|.{{GPUTextureDescriptor/sampleCount}}.
-                    - |srcTextureDesc|.{{GPUTextureDescriptor/format}} is equal to |dstTextureDesc|.{{GPUTextureDescriptor/format}}.
+                    - If |srcTextureDesc|.{{GPUTextureDescriptor/format}} is not equal to |dstTextureDesc|.{{GPUTextureDescriptor/format}}:
+                        - the copy configuration is allowed [=copies of depth and stencil textures=]
```

Comment:
```suggestion
                        - the [=copies of depth and stencil textures|copy configuration is allowed=]
```

"the copy configuration is allowed copies of depth and stencil textures" doesn't read very well to me, so I'd suggest making the "copy configuration is allowed" the link text.

Alternatively, you could write something like:

> - the copy configuration is allowed _as defined by_ [=copies of depth and stencil textures=]